src/dripper/DripRequestHandler.ts

```diff
@@ -20,22 +21,26 @@ const isParachainValid = (parachain: string): boolean => {
   return id > 999 && id < 10_000;
 };
 
+type HandleRequestOpts =
+  | ({ external: true; captchaResponse: string } & Omit<DripRequestType, "sender">)
+  | ({ external: false; sender: string } & Omit<DripRequestType, "captchaResponse">);
+
 export class DripRequestHandler {
   constructor(
     private actions: PolkadotActions,
-    private recaptcha: Recaptcha,
+    private captchaService: Procaptcha | Recaptcha,
   ) {}
 
-  async handleRequest(
-    opts:
-      | ({ external: true; recaptcha: string } & Omit<DripRequestType, "sender">)
-      | ({ external: false; sender: string } & Omit<DripRequestType, "recaptcha">),
-  ): Promise<DripResponse> {
+  async handleRequest(opts: HandleRequestOpts): Promise<DripResponse> {
     const { external, address: addr, parachain_id, amount } = opts;
     counters.totalRequests.inc();
 
-    if (external && !(await this.recaptcha.validate(opts.recaptcha)))
-      return { error: "Captcha validation was unsuccessful" };
+    if (external) {
+      const captchaValidate = await this.captchaService.validate(opts.captchaResponse);
+      console.log("Captcha validate response", JSON.stringify(captchaValidate));
```

Comment:
Could you use the logger from `src/logger.ts` please?

```suggestion
      logger.debug(`Captcha validate response ${JSON.stringify(captchaValidate)}`);
```